docs/hpc/03_storage/06_data_management.mdx

```diff
@@ -0,0 +1,115 @@
+
+In addition, a number of ***data management tools*** are available that enable data transfers and  data sharing, recommended best practices, and various scenarios and use cases of managing data in the HPC Environment.
+
+Multiple ***public data sets*** are available to all users of the HPC environment, such as a subset of The Cancer Genome Atlas (TCGA), the Million Song Database, ImageNet, and Reference Genomes. 
+
+Below is a list of file systems with their characteristics and a summary table. Reviewing the list of available file systems and the various Scenarios/Use cases that are presented below, can help select the right file systems for a research project. As always, if you have any questions about data storage in the HPC environment, you can request a consultation with the HPC team by sending email to [hpc@nyu.edu](mailto:hpc@nyu.edu).
+
+## Data Security Warning
+### Moderate Risk Data - HPC Approved
+-   The HPC Environment has been approved for storing and analyzing **Moderate Risk research data**, as defined in the [NYU Electronic Data and System Risk Classification Policy](https://www.nyu.edu/about/policies-guidelines-compliance/policies-and-guidelines/electronic-data-and-system-risk-classification.html). 
+-   **High Risk** research data, such as those that include Personal Identifiable Information (**PII**) or electronic Protected Health Information (**ePHI**) or Controlled Unclassified Information (**CUI**) **should NOT be stored in the HPC Environment**. 
+:::note
+only the Office of Sponsored Projects (OSP) and Global Office of Information Security (GOIS) are empowered to classify the risk categories of data.
+:::
+
+### High Risk Data - Secure Research Data Environments (SRDE) Approved
+Because the HPC system is not approved for High Risk data, we recommend using an approved system like the  [Secure Research Data Environments (SRDE)](../../srde/01_getting_started/01_intro.md).
+
+## Data Storage options in the HPC Environment
+### User Home Directories
+Every individual user has a home directory (under **`/home/$USER`**, environment variable **`$HOME`**) for permanently storing code and important configuration files. Home Directories provide limited storage space (**50 GB**) and inodes (files) **30,000** per user. Users can check their quota utilization using the [myquota](http://www.info-ren.org/projects/ckp/tech/software/version/myquota.html) command.
+
+User home directories are backed up daily and old files under **`$HOME`** are not purged.
+
+The User home directories are available on all HPC clusters (Greene) and on every cluster node (login nodes, compute nodes) as well as and Data Transfer Node (gDTN). 
+
+:::note
+-   **Avoid changing file and directory permissions in your home directory to allow other users to access files.** User Home Directories are not ideal for sharing files and folders with other users. HPC Scratch of Research Project Space (RPS) are better file systems for sharing data.
```

Comment:
It is possible to add nested-admonitions to create emphasis : https://docusaurus.io/docs/markdown-features/admonitions#nested-admonitions

Maybe convert the first one to a warning?

Reply:
Oh, cool, I didn't realize I could nest those.  Thanks!
Actually, I had trouble getting that to work inside the list.
Reading over it again I see that the first bullet point is about permissions on home directories and the other 3 are about running out of space.  I've pulled the first point out of the list and note and made the warning before it.
Please take a look at the updated version from my last commit and let me know if this looks ok.  Thanks!

Reply:
Thanks so much for your example.  That really helps.
Actually, there are 3 places in the doc where the first bullet point really defines a warning and then the remaining points are tips, so I changed them all to this style.
I'm happy to change it if you'd prefer, but I'm going to resolve this and start comments on the updated documents below.